### ISSUE TITLE
Missing jack_disconnect.exe

### ISSUE THREAD

**Issue by linuxgeek** [jackaudio/jack2#797]
Missing jack_disconnect.exe
After reading thru the code, I can see that the name of the executable determines how it behaves. So copying jack_connect.exe to jack_disconnect.exe works. Probably should add copying the file in the installer.

## Environment
Windows

## Steps To Reproduce

**Comment by falkTX** [comment]
Good point, this is more due to the fact that Windows doesn't have proper symlink support.
So a copy should be done instead of nothing.

**Comment by linuxgeek** [comment]
I thought that was probably the reason. thx
